pages/index.js

```diff
@@ -4,34 +4,44 @@ import Header from "@/src/components/Header/Header";
 import Country from "@/src/components/Country/Country";
 import { countries } from "@/lib/db";
 
-const Stylemain = styled.main`
+const StyleSection = styled.section`
   margin-top: 7rem;
   background-color: #cbf3f0;
 `;
-
+const StyleList = styled.li`
+  list-style: none;
+  display: flex;
+  gap: 2rem;
+`;
 export default function Countries() {
   return (
-    <Stylemain>
+    <>
       <Header title="All Countries" />
-      <div className="country">
-        {countries.map((country) => {
-          return (
-            <ul key={countries.id}>
-              <Country
-                nameOfCountry={country.name}
-                codeOfcountry={country.code}
-              />
-              <ol>
-                {country.cities.map((city) => (
-                  <ol key={city}>
-                    <Link href={`/DetailsTrip/`}>{city}</Link>
-                  </ol>
-                ))}
-              </ol>
-            </ul>
-          );
-        })}
-      </div>
-    </Stylemain>
+      <StyleSection className="country">
+        <ul>
+          {countries.map((country, index) => {
+            return (
+              <li key={index}>
```

Comment:
Doing the key with index might be a bad practice, you will run into issues when your countries array starts mutating.